### PR TITLE
Updates for namespace section

### PR DESCRIPTION
**Pull request by aarongundel**
Form based editing of the namespace name (non-localized string)

### CODE REVIEW

File: arches_lingo/pkg/graphs/resource_models/Scheme.json
Comment (chrabyrd) [inline arches_lingo/pkg/graphs/resource_models/Scheme.json:1]: I'm assuming Rob's ok with the model change, yeah?

File: arches_lingo/src/arches_lingo/components/generic/ControlledListItem.vue
Comment (chrabyrd) [inline arches_lingo/src/arches_lingo/components/generic/ControlledListItem.vue:2]: yeah please roll back the declaring of emit values as constants. Much rather _not_ have the two script tags

File: arches_lingo/src/arches_lingo/components/generic/ControlledListItem.vue
Comment (chrabyrd) [inline arches_lingo/src/arches_lingo/components/generic/ControlledListItem.vue:17]: I would, however, recommend declaring `view` and `edit` as consts

File: arches_lingo/src/arches_lingo/components/generic/NonLocalizedString.vue
Comment (chrabyrd) [inline arches_lingo/src/arches_lingo/components/generic/NonLocalizedString.vue:2]: same. please roll back the string-as-const so there's not two script tags.

File: arches_lingo/src/arches_lingo/components/generic/NonLocalizedString.vue
Comment (chrabyrd) [inline arches_lingo/src/arches_lingo/components/generic/NonLocalizedString.vue:1]: This looks almost identical to `ControlledListItem.vue`. Is there path toward genericizing this commonality?

File: arches_lingo/src/arches_lingo/components/generic/NonLocalizedStringEditor.vue
Comment (chrabyrd) [inline arches_lingo/src/arches_lingo/components/generic/NonLocalizedStringEditor.vue:8]: styling nit: I'd expect newlines between props, declared consts, emits, and `watch`es . However I won't block on this til we flesh out the styleguide to include it 😄

File: arches_lingo/src/arches_lingo/components/generic/NonLocalizedStringViewer.vue
Comment (chrabyrd) [inline arches_lingo/src/arches_lingo/components/generic/NonLocalizedStringViewer.vue:5]: should this be a p-element? idk this may warrant larger discussion. What would we expect the read-only element to be in the case of _every_ widget? In my head for this one it's a disabled input, but I do not feel strongly about this. @robgaston ?

File: arches_lingo/src/arches_lingo/components/scheme/editor/SchemeEditor.vue
Comment (chrabyrd) [inline arches_lingo/src/arches_lingo/components/scheme/editor/SchemeEditor.vue:47]: I'm typically against single-letter declarations as it makes the code harder to read.

File: arches_lingo/src/arches_lingo/components/scheme/editor/SchemeEditor.vue
Comment (chrabyrd) [inline arches_lingo/src/arches_lingo/components/scheme/editor/SchemeEditor.vue:48]: declare `edit` as a const?

File: arches_lingo/src/arches_lingo/components/scheme/editor/SchemeEditor.vue
Comment (chrabyrd) [inline arches_lingo/src/arches_lingo/components/scheme/editor/SchemeEditor.vue:31]: these _may_ not need to be part of the component config if we expect every component to have the same props/`on` logic

File: arches_lingo/src/arches_lingo/components/scheme/report/SchemeNamespace.vue
Comment (chrabyrd) [inline arches_lingo/src/arches_lingo/components/scheme/report/SchemeNamespace.vue:47]: I'd say declare all functions with `function foo()` declarations, and put them on the bottom of the script tag so they're not mixed in with `onMounted` and `defineExpose`. There's an ordering of stuff in the script tag that I haven't promulgated yet, but for the time being IMO we should still be writing for readability.

File: arches_lingo/src/arches_lingo/constants.ts
Comment (chrabyrd) [inline arches_lingo/src/arches_lingo/constants.ts:2]: <3

**Review by chrabyrd — CHANGES_REQUESTED** [review]
overall looking good! Just a few minor things 👍

File: arches_lingo/pkg/graphs/resource_models/Scheme.json
Comment (aarongundel) [reply arches_lingo/pkg/graphs/resource_models/Scheme.json:1]: we discussed this in a lingo standup, but I'll cc @robgaston just to make sure he's aware.

File: arches_lingo/src/arches_lingo/components/generic/NonLocalizedStringEditor.vue
Comment (aarongundel) [reply arches_lingo/src/arches_lingo/components/generic/NonLocalizedStringEditor.vue:8]: better now than later!

File: arches_lingo/src/arches_lingo/components/generic/NonLocalizedStringViewer.vue
Comment (aarongundel) [reply arches_lingo/src/arches_lingo/components/generic/NonLocalizedStringViewer.vue:5]: IMO a non-form element will look better in a report, but have no particular preference - I'll defer to consensus.

File: arches_lingo/src/arches_lingo/components/scheme/report/SchemeNamespace.vue
Comment (aarongundel) [reply arches_lingo/src/arches_lingo/components/scheme/report/SchemeNamespace.vue:47]: fair enough

File: arches_lingo/src/arches_lingo/components/scheme/editor/SchemeEditor.vue
Comment (aarongundel) [reply arches_lingo/src/arches_lingo/components/scheme/editor/SchemeEditor.vue:31]: fair, I'll keep it simple till we need otherwise.

File: arches_lingo/pkg/graphs/resource_models/Scheme.json
Comment (robgaston) [reply arches_lingo/pkg/graphs/resource_models/Scheme.json:1]: yup, all good!

File: arches_lingo/src/arches_lingo/components/generic/NonLocalizedString.vue
Comment (aarongundel) [reply arches_lingo/src/arches_lingo/components/generic/NonLocalizedString.vue:1]: One will likely develop as a result of #129

**Review by chrabyrd — CHANGES_REQUESTED** [review]
looking good, just a few smaller things now

**Review by chrabyrd — APPROVED** [review]
lgtm! 🚀

**Review by chrabyrd — APPROVED** [review]
lgtm! 🚀

**Pull request merged**
